swagger.yml

```
@@ -0,0 +1,1098 @@
+      - vorlaufzinsenProTag
+
+  versicherungskonditionen:
+    type: object
+    properties:
+      basisFuerDieProvisionsermittlung:
+        type: string
+        enum:
+          - VERSICHERUNGS_PRAEMIE_NETTO
+          - VERSICHERUNGS_PRAEMIE_BRUTTO
+      versicherungsprovision:
+        $ref: '#/definitions/euro'
+      versicherungspraemieBrutto:
+        description: Versicherungsprämie inklusive Steuer für Arbeitslosigkeitsversicherung
+        $ref: '#/definitions/euro'
+      versicherungspraemieNetto:
+        description: Versicherungsprämie exklusive Steuer für Arbeitslosigkeitsversicherung
+        $ref: '#/definitions/euro'
+      versicherungsrisiken:
+        type: array
+        items:
+          $ref: '#/definitions/versicherungsrisiken'
+      versicherer:
+        type: string
+    required:
+      - basisFuerDieProvisionsermittlung
+      - versicherungsprovision
+      - versicherungspraemieBrutto
+      - versicherungspraemieNetto
+      - versicherungsrisiken
+      - versicherer
+
+  meldung:
+    type: object
+    properties:
+      kategorie:
+        type: string
+        enum:
+          - MACHBARKEIT
+          - VOLLSTAENDIGKEIT
+          - HINWEIS
+          - ANPASSUNG
+      text:
+        type: string
+      code:
+        type: string
+    required:
+      - kategorie
+      - text
+      - code
+
+
+  bonitaetscheckPosition:
+    type: object
+    properties:
+      positionen:
+        type: array
+        items:
+          type: object
+          properties:
+            label:
+              type: string
+            wert:
+              $ref: '#/definitions/euro'
+          required:
+            - label
+            - wert
+    required:
+      - positionen
+
+  dokument:
+    type: object
+    properties:
+      bezeichnung:
+        type: string
+      dateiname:
+        type: string
+      base64pdf:
+        type: string
+      sichtbarkeit:
+        type: object
+        properties:
+          sichtbarFuerVertrieb:
+            type: boolean
+          sichtbarFuerProduktanbieter:
+            type: boolean
+    required:
+      - bezeichnung
+      - dateiname
+      - base64pdf
+      - sichtbarkeit
+
+###########################################
+### Globale Typen fuer Request+Response ###
+###########################################
+
+
+  versicherungsrisiken:
+    type: object
+    properties:
+      antragstellerId:
+        type: string
+      tod:
+        type: boolean
+      arbeitsunfaehigkeit:
+        type: boolean
+      arbeitslosigkeit:
+        type: boolean
+    required:
+      - antragstellerId
+      - tod
+      - arbeitsunfaehigkeit
+      - arbeitslosigkeit
+
+  euro:
+    type: object
+    description: Euro-Betrag mit bis zu 2 Nachkommastellen
+    properties:
+      value:
+        type: number
+        format: 'xxx.xx'
+
+  prozent:
+    type: object
+    description: 100 basierter Prozentwert mit bis zu 2 Nachkommastellen
```

Prozentwerte sind in unseren bisherigen APIs der ME immer 1 basiert

```suggestion
    description: 1 basierter Prozentwert mit bis zu 4 Nachkommastellen
```